docs/resources/model_serving.md

```diff
@@ -45,32 +45,75 @@ The following arguments are supported:
 
 * `name` - (Required) The name of the model serving endpoint. This field is required and must be unique across a workspace. An endpoint name can consist of alphanumeric characters, dashes, and underscores. NOTE: Changing this name will delete the existing endpoint and create a new endpoint with the update name.
 * `config` - (Required) The model serving endpoint configuration.
-
-### config Configuration Block
-
-* `served_models` - (Required) Each block represents a served model for the endpoint to serve. A model serving endpoint can have up to 10 served models.
-* `traffic_config` - A single block represents the traffic split configuration amongst the served models.
-
-### served_models Configuration Block
+  * `served_entities` - A list of served entities for the endpoint to serve. A serving endpoint can have up to 10 served entities.
+  * `served_models` - (Deprecated, use `served_entities` instead) Each block represents a served model for the endpoint to serve. A model serving endpoint can have up to 10 served models.
+  * `traffic_config` - A single block represents the traffic split configuration amongst the served models.
+    * `routes` - (Required) Each block represents a route that defines traffic to each served model. Each `served_models` block needs to have a corresponding `routes` block
+        * `served_model_name` - (Required) The name of the served model this route configures traffic for. This needs to match the name of a `served_models` block
+        * `traffic_percentage` - (Required) The percentage of endpoint traffic to send to this route. It must be an integer between 0 and 100 inclusive.
+  * `auto_capture_config` - Configuration for Inference Tables which automatically logs requests and responses to Unity Catalog.
+    * `catalog_name` - The name of the catalog in Unity Catalog. NOTE: On update, you cannot change the catalog name if it was already set.
+    * `schema_name` - The name of the schema in Unity Catalog. NOTE: On update, you cannot change the schema name if it was already set.
+    * `table_name_prefix` - The prefix of the table in Unity Catalog. NOTE: On update, you cannot change the prefix name if it was already set.
+    * `enabled` - If inference tables are enabled or not. NOTE: If you have already disabled payload logging once, you cannot enable again.
+* `tags` - Tags to be attached to the serving endpoint and automatically propagated to billing logs.
+* `rate_limits` - Rate limits to be applied to the serving endpoint. NOTE: only external and foundation model endpoints are supported as of now.
```

Comment:
Can we add a block in the documents expanding on the structure for `rate_limits` as well?